std/conv.d

```diff
@@ -1324,7 +1333,7 @@ in
 {
     assert(radix >= 2 && radix <= 36);
 }
-do
+body
```

Comment:
Huh?

Reply:
It looks like potentially @Dechcaudron used a stale version of the repository to base his changes from?

Reply:
Hmm git is smart enough for this (it thinks in hunks/changes, not files). It looks more like a rebase with mistakes or copy/pasting an old version of this file into a up-to-date branch.

Reply:
@wilzbach's assesment is accurate. I fucked up real bad with this one. I'll fix it and let you guys know.